ui/util/formatAriaLabel.js

```diff
@@ -0,0 +1,32 @@
+import DateTime from 'component/dateTime';
+
+export function formatClaimPreviewTitle(title, channelTitle, date, mediaDuration) {
+  // Aria-label value for claim preview
+  let ariaDate = date ? DateTime.getTimeAgoStr(date) : null;
+  let ariaLabelData = title;
+
+  if (mediaDuration) {
+    if (ariaDate) {
+      ariaLabelData += ariaLabelData = __('%title% by %channelTitle% %ariaDate%, %mediaDuration%', {
+        title,
+        channelTitle,
+        ariaDate,
+        mediaDuration,
+      });
+    } else {
+      ariaLabelData += ariaLabelData = __('%title% by %channelTitle%, %mediaDuration%', {
+        title,
+        channelTitle,
+        mediaDuration,
+      });
+    }
+  } else {
+    if (ariaDate) {
+      ariaLabelData += ariaLabelData = __('%title% by %channelTitle% %ariaDate%', { title, channelTitle, ariaDate });
+    } else {
+      ariaLabelData += ariaLabelData = __('%title% by %channelTitle%', { title, channelTitle });
+    }
+  }
+
+  return ariaLabelData;
```

Comment:
I think we forgot to remove the `+=`?

<img src="https://user-images.githubusercontent.com/64950861/127447038-08f215ba-663f-4670-bdab-2736eca4215f.png" width="400">